delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/plugin/PluginDef.scala

```diff
@@ -11,17 +12,34 @@ trait PluginDef {
     */
   def module: ModuleDef
 
+  /**
+    * Remote context resolution provided by the plugin.
+    */
+  def remoteContextResolution: RemoteContextResolution
+
   /**
     * Plugin information
     */
   def info: PluginInfo
 
+  /**
+    * The priority of this plugin.
+    * This value will decide the order in which this plugin takes is executed compared to the rest of the plugins.
+    * It affects Routes ordering and classpath ordering
+    */
+  def priority: Int
```

Comment:
If we add priority, we can't handle it in the code, we will have to handle them via configuration as it really depends on which plugins are here

Reply:
Priority is now read from configuration present in each plugin and it can be overridden by system properties